doc/source/getting_started/10min.rst

```diff
@@ -83,6 +83,13 @@ As you can see, the columns ``A``, ``B``, ``C``, and ``D`` are automatically
 tab completed. ``E`` is there as well; the rest of the attributes have been
 truncated for brevity.
 
+.. note::
```

Comment:
this doesn't seem like a good place for this, can you put near usages of concat instead

Reply:
Sure, even I felt that would be a better place for this.